testsuites/syncgateway/functional/tests/conftest.py

```diff
@@ -168,6 +168,24 @@ def params_from_base_suite_setup(request):
         log_info("Using document storage for sync meta data")
         persist_cluster_config_environment_prop(cluster_config, 'xattrs_enabled', False)
 
+    try:
+        server_version
```

Comment:
Not sure if this is needed. If server_version is not specified, it will blow up at line 117: check_xattr_support(server_version, sync_gateway_version). We always specify server_version. I don't understand the motivation behind adding server_version: "", to the cluster config.